### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by wendigo**
## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -457,7 +457,7 @@
             <dependency>
                 <groupId>com.clickhouse</groupId>
                 <artifactId>clickhouse-jdbc</artifactId>
-                <version>0.6.0-patch4</version>
+                <version>0.6.0-patch5</version>
```
Comment (ebyhr) [inline pom.xml:460]: This version and `partch4` are still pre-release. I think we shouldn't use it. 
https://github.com/ClickHouse/clickhouse-java/releases/tag/v0.6.0-patch5

**Review by ebyhr — APPROVED** [review]
Looks good except for ClickHouse bump.

File: pom.xml (hunk 1 above)
Comment (wendigo) [reply pom.xml:460]: As far as I understand the project structure these are patch releases to 0.6.0 release.

Comment (ebyhr) [reply pom.xml:460]: I know they are patch releases. However, `patch3` is the latest release on GitHub. Is it really safe to use 4 and 5? 
Screenshot 2024-05-28 at 7 20 17

**Pull request merged**